### PR TITLE
chore: update bug template

### PR DESCRIPTION
**Pull request by edwardfoyle**
#### Description of changes

Update repro steps description

#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by lazpavel** [comment]
LGTM

**Review by lazpavel — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/1.bug_report.yaml [hunk 1]
```diff
@@ -4,13 +4,14 @@ description: Create a report to help us improve Amplify CLI
...
-      value: | 
+      value: |
         Thanks for taking the time to fill out this bug report! Try to include as much information as you can.
 
         For reference, the current version of the Amplify CLI is <img src="https://img.shields.io/npm/v/@aws-amplify/cli.svg" />.
 
-        **Note**: If your issue is regarding the AWS Amplify Console service, please log it in the [Amplify Console GitHub Issue Tracker](https://github.com/aws-amplify/amplify-console/issues).
- 
+        > **For Amplify Console issues, go [here](https://github.com/aws-amplify/amplify-console/issues/new/choose).**
+        > **For Amplify codegen issues, go [here](https://github.com/aws-amplify/amplify-codegen/issues/new/choose).**
```
Comment (siegerts) [inline .github/ISSUE_TEMPLATE/1.bug_report.yaml:13]:
```suggestion
        > For **Amplify Codegen** issues, go [here](https://github.com/aws-amplify/amplify-codegen/issues/new/choose).
```

File: .github/ISSUE_TEMPLATE/1.bug_report.yaml [hunk 2]
```diff
@@ -4,13 +4,14 @@ description: Create a report to help us improve Amplify CLI
...
     attributes:
-      value: | 
+      value: |
         Thanks for taking the time to fill out this bug report! Try to include as much information as you can.
 
         For reference, the current version of the Amplify CLI is <img src="https://img.shields.io/npm/v/@aws-amplify/cli.svg" />.
 
-        **Note**: If your issue is regarding the AWS Amplify Console service, please log it in the [Amplify Console GitHub Issue Tracker](https://github.com/aws-amplify/amplify-console/issues).
- 
+        > **For Amplify Console issues, go [here](https://github.com/aws-amplify/amplify-console/issues/new/choose).**
```
Comment (siegerts) [inline .github/ISSUE_TEMPLATE/1.bug_report.yaml:12]:
```suggestion
        > For **Amplify Console** issues, go [here](https://github.com/aws-amplify/amplify-console/issues/new/choose).
```

File: .github/ISSUE_TEMPLATE/1.bug_report.yaml [hunk 3]
```diff
@@ -109,10 +110,11 @@ body:
     attributes:
       label: Reproduction steps
       description: |
-        How do you trigger this bug? Please walk us through it step by step. 
+        How can this bug be reproduced?
+        As a general guideline, ask yourself: "will running the exact steps below in an empty directory create an Amplify project that demonstrates the bug behavior?"
```
Comment (siegerts) [inline .github/ISSUE_TEMPLATE/1.bug_report.yaml:114]: Thoughts on removing the _guideline_ part- 


How can this bug be reproduced? Will running the **exact** steps below _in an empty directory_ create an Amplify project that demonstrates the bug behavior?

Comment (edwardfoyle) [reply .github/ISSUE_TEMPLATE/1.bug_report.yaml:114]:
```suggestion
        Will running the **exact** steps below _in an empty directory_ create an Amplify project that demonstrates the bug behavior?"
```

**Review by siegerts — APPROVED** [review]

**Pull request merged**